ingest/rules/fetch_from_ncbi.smk

```diff
@@ -48,26 +21,28 @@ rule fetch_ncbi_dataset_package:
     benchmark:
         "benchmarks/fetch_ncbi_dataset_package.txt"
     shell:
-        # what's the `:q` mean
         """
         datasets download virus genome taxon {params.ncbi_taxon_id:q} \
             --no-progressbar \
             --filename {output.dataset_package}
         """
 
 
-# Note: This rule is not part of the default workflow!
-# It is intended to be used as a specific target for users to be able
-# to inspect and explore the full raw metadata from NCBI Datasets.
+# Note: This rule is not part of the default workflow! It is intended
+# to be used as a specific target to be able to inspect and explore
+# the full raw metadata from NCBI Datasets.
 rule dump_ncbi_dataset_report:
```

Comment:
_non-blocking_

I think fair to remove this rule if you are past the data exploration phase for ingest.

Reply:
I think until I'm positive I've got all the metadata for the phylo build, I'll leave this alone for the moment — will try to remember to loop back once the phylo build is more settled.